sources/models/shared-defaults/eth0-sysctls.toml

```diff
@@ -0,0 +1,5 @@
+[settings.kernel.sysctl]
+# Accept router advertisement (RA) packets even if IPv6 forwarding is enabled on eth0
+"net.ipv6.conf.eth0.accept_ra" = "2"
+# Enable loose mode for reverse path filter
+"net.ipv4.conf.eth0.rp_filter" = "2"
```

Comment:
No migration is needed because these defaults will be applied to existing hosts on upgrade? It's worth testing that specifically.